antora-playbook.yml

```diff
@@ -42,6 +42,7 @@ content:
       branches:
         - "3.4"
         - "3.5"
+        - "3.6"
```

Comment:
❓ Is this version the new latest version? If so, we should also update the redirect in the `netlify.toml` file.

Reply:
as discussed with @uguy this is the latest version, so I am going to update the redirect

Reply:
done in 743f575